src/Sarif/VersionOne/SarifUtilitiesVersionOne.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.CodeAnalysis.Sarif.VersionOne;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public static class SarifUtilitiesVersionOne
+    {
+        private const string V1_0_0 = "1.0.0";
+
+        public static string ConvertToText(this SarifVersionVersionOne sarifVersion)
+        {
+            switch (sarifVersion)
+            {
+                case SarifVersionVersionOne.OneZeroZero: { return V1_0_0; }
+            }
+            throw new ArgumentException("Unsupported version", nameof(sarifVersion));
```

Comment:
I'd put this as the `default` case of the `switch`, to demonstrate that you've covered all cases. Some analyzers will complain if your `switch` doesn't cover every case.

Reply:
ok